sql/tsql/TSqlLexer.g4

```diff
@@ -1053,6 +1066,12 @@ BINARY:              '0' 'X' HEX_DIGIT*;
 FLOAT:               DEC_DOT_DEC;
 REAL:                (DECIMAL | DEC_DOT_DEC) ('E' [+-]? DEC_DIGIT+);
 
+fragment IPV4_OCTET:          DEC_DIGIT DEC_DIGIT? | [0-2] DEC_DIGIT DEC_DIGIT;
+fragment IPV6_QUARTET:        HEX_DIGIT (HEX_DIGIT (HEX_DIGIT HEX_DIGIT?)?)?;
+
+IPV4_ADDR:           IPV4_OCTET DOT IPV4_OCTET DOT IPV4_OCTET DOT IPV4_OCTET;
+IPV6_ADDR:           IPV6_QUARTET? COLON IPV6_QUARTET? COLON IPV6_QUARTET? COLON IPV6_QUARTET? COLON IPV6_QUARTET? COLON IPV6_QUARTET? COLON IPV6_QUARTET? COLON IPV6_QUARTET?;
```

Comment:
Could you please elaborate on the need of these rules? I see neither parse rules that use them nor test samples.

Reply:
I originally noticed on the MS SQL Server documentation there was _something_ (I can't find it anymore) that needed an IPv4/IPv6 address. But it looks like, as you said, I never made the corresponding changes to the parser rules.

I'll just remove these for now. If I eventually find the statements that have the IP addresses we can just do a separate PR for it. It doesn't seem to be related to `SPARSE`.